components/combobox/__examples__/dialog.jsx

```diff
@@ -35,9 +35,9 @@ class Example extends React.Component {
 		return inputValue;
 	};
 
-	handleCheckboxChange(target, value) {
-		const { checked } = this.state;
-		if (target.checked) {
+	handleCheckboxChange(targetChecked, target, value) {
+		const checked = this.state.checked;
```

Comment:
Please use the object destructuring syntax here. This will also pass the lint issue in travis CI